geniza/corpus/models.py

```diff
@@ -264,6 +264,7 @@ class Document(ModelIndexable):
     appears on one or more fragments."""
 
     id = models.AutoField("PGPID", primary_key=True)
+    old_pgpids = models.CharField(max_length=255, blank=True)
```

Comment:
could we use a postgres arrayfield with integers here? https://docs.djangoproject.com/en/3.2/ref/contrib/postgres/fields/#arrayfield